<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>itemDataView</class>
 <widget class="QWidget" name="itemDataView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="iDlabel">
      <property name="text">
       <string>Id</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="iDEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox_backL">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Backlight</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="branlabel">
      <property name="text">
       <string>Brand</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="brandEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox_finger">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fingerprint</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_11">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QLabel" name="cam2label">
      <property name="text">
       <string>Cam2(Megapx)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="cam2Edit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>245</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioSmartwatch">
      <property name="text">
       <string>Smartwatch</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioNotebook">
      <property name="text">
       <string>Notebook</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioSmartphone">
      <property name="text">
       <string>Smartphone</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="displaylabel">
      <property name="text">
       <string>Display(&quot;)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="displayEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="cancelbtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color:  rgb(255,0,0);
    border-radius: 3px;
	color: black;
}
QPushButton:pressed {
    background-color: rgb(203, 218, 249);
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Elimina ogg</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="pricelabel">
      <property name="text">
       <string>Price($)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="priceEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Modifica Oggetto</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="memorylabel">
      <property name="text">
       <string>Memory(Gb)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="memoryEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="modellabel">
      <property name="text">
       <string>Model</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="modelEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox_fanles">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Fanless</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_9">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="GPUlabel">
      <property name="text">
       <string>GPU</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="GPUEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_10">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="cam1label">
      <property name="text">
       <string>Cam1(Megapx)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="cam1Edit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="processorlabel">
      <property name="text">
       <string>Processor</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="processorEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_15">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_18">
    <item>
     <widget class="QLabel" name="sensorlabel">
      <property name="text">
       <string>Sensor</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="sensorEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_14">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_17">
    <item>
     <widget class="QLabel" name="materiallabel">
      <property name="text">
       <string>MaterialC</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="materialEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_13">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_16">
    <item>
     <widget class="QLabel" name="certificatelabel">
      <property name="text">
       <string>Certificate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="certificateEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="confirmbtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>430</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color:  rgb(255,0,0);
    border-radius: 3px;
	color: black;
}
QPushButton:pressed {
    background-color: rgb(203, 218, 249);
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Conferma</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
